.env.example

```diff
@@ -2,7 +2,7 @@ STRIPE_SECRET_KEY = "sk_test_51JJ3vNLOlCyqSHKrp2vu93c9YzrXxDq7otphDYZqQTE10eZ0Gs
 CANCEL_URL = "https://helpafamily.margaritahumanitarian.org/"
 SUCCESS_URL = "https://helpafamily.margaritahumanitarian.org/thank-you"
 NEXT_PUBLIC_GMAPS_JS_API_KEY = "<your Google Maps JavaScript API Key here>"
-
+REACT_APP_SITE_KEY='6LeSv6ccAAAAAGL98cTakxdX_lvAUe-D3SR06bXw'
```

Comment:
Oops!! it seems your API key is here please take care .

Reply:
same here